### PR TITLE
Add Lodestar to mobile app client monitoring setup guides

### PR DESCRIPTION
**Pull request by nflaig**
Adds lodestar to mobile app client monitoring setup guides.

I tried to align it with how Lighthouse is documented, as technically their setup is the same.

I did not find a quick way to properly test this, so might be worth to check if this renders as expected.

### CODE REVIEW

**Comment by nflaig** [comment]
Are the quality checks in place to ensure that clients you are listing on your website provide accurate data?

I verified that Lodestar works correctly but would be great if this was additionally confirmed by you.

**Comment by Buttaa** [comment]
> Are the quality checks in place to ensure that clients you are listing on your website provide accurate data?

No, we only display whatever the client sends, so it's totally up to the user/client

**Comment by manuelsc** [comment]
@nflaig Thank you very much for your contribution! I pushed a quick fix to the repo that prevents lighthouse and lodestar to simultaneously open. We used a bad naming scheme there that was easy to misuse, that was on us.

LGTM

**Review by manuelsc — APPROVED** [review]

File: templates/user/settings.html [hunk 1]
```diff
@@ -626,6 +630,29 @@ <h2 class="card-header" id="headingFour">
                             </div>
                           </div>
                         </div>
+                        <div class="card" style="box-shadow: none; ">
+                          <h2 class="card-header" id="headingOne">
```
Comment (nflaig) [inline templates/user/settings.html:634]: @manuelsc Thanks for the update. Not sure how relevant this is for the id here but `headingOne` is the same as for Lighthouse

Comment (manuelsc) [reply templates/user/settings.html:634]: Good catch I'll raise a ticket internally to clean this up too. Seems to only have a descriptive impact

**Pull request merged**